rosetta-lang/src/main/java/com/regnosys/rosetta/formatting2/XtextResourceFormatter.java

```diff
@@ -47,30 +45,39 @@ public void formatCollection(Collection<Resource> resources, ITypedPreferenceVal
 	}
 
 	@Override
-	public void formatXtextResource(XtextResource resource, ITypedPreferenceValues preferenceValues) {
+	public void formatXtextResource(XtextResource resource, ITypedPreferenceValues preferenceValues,
+			BiConsumer<Resource, String> handler) {
+		LOGGER.info("Formatting file at location " + resource.getURI());
+
 		// setup request and formatter
 		FormatterRequest req = formatterRequestProvider.get();
 		req.setPreferences(preferenceValues);
 		IFormatter2 formatter = iFormatter2Provider.get();
 
-		ITextRegionAccess regionAccess = regionBuilder.forNodeModel(resource).create();
+		ITextRegionAccess regionAccess = null;
+		try {
+			regionAccess = regionBuilder.forNodeModel(resource).create();
+		} catch (Exception e) {
+			LOGGER.info("Resource " + resource.getURI() + " is empty.", e);
```

Comment:
About this error message: are you sure that this will only occur if the resource is empty? Since you're capturing a general `Exception` here.